ansible/roles/docker/vars/main.yml

```diff
@@ -2,3 +2,4 @@
 # role variable for the swarm service
 
 docker_api_port: 2385
+thin_provisioned: false 
```

Comment:
sure, we can leave this in vars. I think we need to cleanup `docker_api_port: 2385` definition then, I think it has slipped in due to merge.